src/insights/position.ts

```diff
@@ -21,4 +22,100 @@ export interface LeaderboardResult {
   meta: PageMeta;
 }
 
-export type QueryGetLeaderboardResponse = Entries<LeaderboardResult>
\ No newline at end of file
+export type QueryGetLeaderboardResponse = Entries<LeaderboardResult>
+
+//Liquidation And ADL
+export interface GetLiquidationAndADLQueryParams extends QueryByPageRequest {
+  sort?: "ASC" | "DESC";
```

Comment:
Since `sort` is defined a few times in this file, might as well create an extended interface to add this
i.e.
`export interface QueryByPageRequestSort extends QueryByPageRequest { ... }`